examples/get-started/hook/controls.jsx

```diff
@@ -4,14 +4,38 @@ import {useCallback, useState, useEffect} from 'react';
 import {useMap} from 'react-map-gl';
 
 export default function Controls() {
-  const {mymap} = useMap();
+  /**
+   * ## This is how `useMap` works:
+   * ```
+   * const demo = useMap();
+   * console.log('Controls', {demo});
+   * ```
+   * ### First render:
+   * ```
+   * {
+   *     "demo": {
+   *         "current": undefined
+   *     }
+   * }
+   * ```
+   * ### Second render:
+   * ```
+   * {
+   *   "demo": {
+   *     "current": undefined,
+   *     "mymap": {...} // See https://visgl.github.io/react-map-gl/docs/api-reference/types#mapref
+   *   }
+   * }
+   * ```
```

Comment:
```suggestion
   * ## This is how `useMap` works:
   * ```
   * const maps = useMap();
   * console.log('Controls useMap()', maps);
   * ```
   * ### First render:
   * ```
   * {
   *     "current": undefined
   * }
   * ```
   * ### Second render:
   * ```
   * {
   *     "current": undefined,
   *     "mymap": {...} // See https://visgl.github.io/react-map-gl/docs/api-reference/types#mapref
   * }
   * ```
```